src/app/pages/signing/signing.component.html

```diff
@@ -1,61 +1,73 @@
-<div class="container py-5 center-children vertical-center">
-  <div class="row">
-    <h2>Sending Message to friend.</h2>
+<div class="container py-5 d-flex vertical-center">
+  <div class="row" id="title-div">
+    <label for="signing-box">
+      <h2 id="signing-header">Type your message to [friend name] below.</h2>
```

Comment:
The `<h2>` tag is not allowed in `<label>`. You should create a CSS class on the `<label>` to format the text.